### PR TITLE
Add "Get Involved" page

### PR DESCRIPTION
**Pull request by TT3rd**
Capture3

After going to the eye doctor last Friday I found out that I have a red-green color deficiency or deuteranopia. But I don't really notice it tho lol.

### CODE REVIEW

**Comment by coramonokandilos** [comment]
Looks great Trent!

A couple of things:

- I think the modules (Art, Volunteer, etc.) should probably be a bit smaller (both height and width)? So we have more white space on the left and right and so we don't have to scroll too much to be able to see all the elements. Right now, the buttons look very small and I think decreasing the space will help with that!
- The "FAQ" title is missing off the FAQs. 
- Make sure you add all the stuff we still need from BAMP in this [sheet](https://docs.google.com/spreadsheets/d/1mqwwYEvk68-PrNebsHkTjBIHoX3_OfuDpcOOrBAuBck/edit#gid=1423163878).

File: get_involved.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+        <section class="pageHeader">
+            <img class="pageHeader--img" src="images/getInvolved/art-class-spray.jpeg">
+            <div class="box box-red">
+                <h1 class="sectionHeading">Get Involved</h1>
+                <p class="pageHeader--text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud...</p>
+            </div>
+        </section>
+
+            <div class="l-section">
```
Comment (coramonokandilos) [inline get_involved.html]: I think there is an indenting issue on this line.

**Review by coramonokandilos — CHANGES_REQUESTED** [review]

**Review by johncip — APPROVED** [review]
looks good. I updated the footer to be the new structure (see https://github.com/MissionBit/BAMPWebsite/pull/63 ). merging

**Comment by johncip** [comment]
@coramonokandilos I think you'll have to approve as well before one of us can merge -- I'm not sure if there was anything else, but the indentation you caught is fixed now.

**Comment by coramonokandilos** [comment]
Looks good to me!

**Review by coramonokandilos — APPROVED** [review]

**Pull request merged**